### PR TITLE
feat: lock dete-picker-view component add

### PR DESCRIPTION
**Pull request by youngjuning**
## Description

lock dete-picker-view component add

## Package

- [x] tuya-panel-lock-sdk

## Type of change

Please select the relevant option.
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by youngjuning** [comment]
LGTM~

File: packages/tuya-panel-lock-sdk/src/date-picker-view/interfance.ts [hunk 1]
```diff
@@ -0,0 +1,240 @@
+import { StyleProp, ViewStyle, ColorPropType, ImageStyle } from 'react-native';
```
Comment (ShinyLeee) [inline packages/tuya-panel-lock-sdk/src/date-picker-view/interfance.ts:1]: 文件名错的,interface.ts 吧

**Review by ShinyLeee — CHANGES_REQUESTED** [review]
文件名修改下

File: packages/tuya-panel-lock-sdk/src/date-picker-view/interfance.ts (hunk 1 above)
Comment (zegezs) [reply packages/tuya-panel-lock-sdk/src/date-picker-view/interfance.ts]: > 文件名错的,interface.ts 吧

Comment (zegezs) [reply packages/tuya-panel-lock-sdk/src/date-picker-view/interfance.ts]: > 文件名错的,interface.ts 吧

已经修复

File: packages/tuya-panel-lock-sdk/src/date-picker-view/i18n/strings.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+export default {
+  en: {
+    placeholderTip: 'Please select date',
```
Comment (ShinyLeee) [inline packages/tuya-panel-lock-sdk/src/date-picker-view/i18n/strings.ts:3]: 多语言 key 命名不规范

**Review by ShinyLeee — CHANGES_REQUESTED** [review]

File: packages/tuya-panel-lock-sdk/src/date-picker-view/i18n/strings.ts (hunk 2 above)
Comment (zegezs) [reply packages/tuya-panel-lock-sdk/src/date-picker-view/i18n/strings.ts]: 已改

**Comment by youngjuning** [comment]
LGTM

**Review by ShinyLeee — APPROVED** [review]
LGTM

**Pull request merged**